tests/System/SystemTestCase.php

```diff
@@ -49,6 +49,7 @@ public function phpbench($command, $workingDir = '.')
     {
         chdir($this->getWorkingDir($workingDir));
         $bin = __DIR__ . '/../../bin/phpbench --verbose';
+        $bin = str_replace('/', DIRECTORY_SEPARATOR, $bin);
```

Comment:
You probably have something slightly different in mind. Functions like `fopen` can handle `/`, but here we are starting a process, nothing related to PHP, only about processes / parameter.